### PR TITLE
provide "size" parameter in torch.normal when called with two floats

### PR DESCRIPTION
**Pull request by umanwizard**
This has been requested in #20323

(It is still not exactly the same as NumPy, which allows you to pass tensors at mean/std and broadcast them with size, but the present PR is extremely simple and does the main thing people are asking for)

### CODE REVIEW

**Comment by umanwizard** [comment]
Ping -- can someone review this?

**Comment by soumith** [comment]
@umanwizard assign it to a reviewer or ping specific people. Most people (or at this point all people) dont read all their github notifications.

**Comment by umanwizard** [comment]
@soumith Whoops, dind't realize I hadn't added anyone

**Comment by fmassa** [comment]
Can you add some documentation for this overload?

**Comment by umanwizard** [comment]
@fmassa ping, can you review again?

**Review by fmassa — APPROVED** [review]
LGTM!

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -3559,6 +3559,22 @@ def merge_dicts(*dicts):
...
+
+.. function:: normal(mean, std, size, *, out=None) -> Tensor
+
+Similar to the function above, but the means and standard deviations are shared
+among all drawn elements. The resulting tensor has size given by :attr:`size`.
+
+Args:
+    mean (float): the mean for all distributions
+    std (float): the standard deviation for all distributions
+    size (int...): a sequence of integers defining the shape of the output tensor.
```
Comment (gchanan) [inline torch/_torch_docs.py]: can we add size everywhere?  That seems consistent and doesn't need yet another overload.

Comment (umanwizard) [reply torch/_torch_docs.py]: We would need the extra overload anyway, to support all of `normal(Tensor, float, sizes)`, `normal(float, Tensor, sizes)`, and `normal(float, float, sizes)`

Comment (gchanan) [reply torch/_torch_docs.py]: can't size have a default parameter?  So you would have only the overloads that exist now.

Comment (umanwizard) [reply torch/_torch_docs.py]: My point is we don't have any overload now that takes two floats. We have overloads that take two tensors, or one tensor and one float.

Comment (gchanan) [reply torch/_torch_docs.py]: sorry I missed that.  I still think we should just add size everywhere, though.

Comment (umanwizard) [reply torch/_torch_docs.py]: For posterity: gchanan and I spoke about this offline. We agreed that we do need to have a new override for the "two floats" case, but there's also no good reason not to add "sizes" everywhere. So I'll do that.

**Comment by zhangguanheng66** [comment]
@gchanan let me know if you have any other questions. It seems ready to land.

**Comment by zhangguanheng66** [comment]
@umanwizard Could you rebase and resolve the merge conflicts? I tried to land the diff and got merge conflict warnings. Thanks.

**Comment by umanwizard** [comment]
Were you able to land this? I will have some free time next week to fix the merge conflicts if still needed

**Comment by zhangguanheng66** [comment]
@umanwizard I have landed the Diff after resolving the merge conflict. No worry for this now.

**Pull request closed without merging**